modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/core.entity_form_display.oe_organisation.oe_stakeholder.default.yml

```diff
@@ -0,0 +1,92 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.oe_organisation.oe_stakeholder.oe_acronym
+    - field.field.oe_organisation.oe_stakeholder.oe_address
+    - field.field.oe_organisation.oe_stakeholder.oe_contact_url
+    - field.field.oe_organisation.oe_stakeholder.oe_logo
+    - field.field.oe_organisation.oe_stakeholder.oe_website
+    - oe_content_entity_organisation.oe_organisation_type.oe_stakeholder
+  module:
+    - address
+    - link
+id: oe_organisation.oe_stakeholder.default
+targetEntityType: oe_organisation
+bundle: oe_stakeholder
+mode: default
+content:
```

Comment:
Please arrange fields so that save button is on bottom

Reply:
Done.